### PR TITLE
first review changes

### PR DESCRIPTION
**Pull request by LockVision**

### CODE REVIEW

File: deliveryMangement-/Customer.java [hunk 1]
```diff
@@ -9,7 +9,7 @@ public Customer(String customerId, String name, String address, String contactNu
         this.customerId = customerId;
         this.name = name;
         this.address = address;
-        this.co = contactNumber;
+        this.contactNumber = contactNumber;
```
Comment (ShabeebReda) [inline deliveryMangement-/Customer.java:12]: great changes

File: deliveryMangement-/Delivery.java [hunk 2]
```diff
@@ -3,10 +3,16 @@
 public class Delivery {
     private String deliveryId;
     private Order order;
-    private String deliveryStatus;  // e.g., "Out for Delivery", "Delivered"
+    private final String deliveryStatus;  // e.g., "Out for Delivery", "Delivered"
```
Comment (ShabeebReda) [inline deliveryMangement-/Delivery.java:6]: well done

File: deliveryMangement-/Customer.java (hunk 1 above)
Comment (LockVision) [reply deliveryMangement-/Customer.java:12]: thank you.

File: deliveryMangement-/Delivery.java (hunk 2 above)
Comment (LockVision) [reply deliveryMangement-/Delivery.java:6]: Thank you.

**Pull request merged**